packages/server/lambdas/alfresco/searchQueryBuilder/luceneQueryBuilder.ts

```diff
@@ -1,54 +1,71 @@
 import { format } from 'date-fns';
-import { IMimeSearchParameter, IModifiedSearchParameter, Paging, SearchParameter } from './types';
+import {
+  IMimeSearchParameter,
+  IModifiedSearchParameter,
+  INameSearchParameter,
+  Paging,
+  SearchParameter,
+  SearchParameterName,
+} from './types';
 
 const mimeTypesMapping = {
+  EXCEL: ['application/vnd.ms-excel', 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'],
```

Comment:
Double quotes inside single quotes like MSWORD and PDF?

Reply:
Updated values and added a test to check for this.